network/stream/v2/syncing_test.go

```diff
@@ -153,7 +153,7 @@ func TestTwoNodesSyncWithGaps(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			sim := simulation.NewBzzInProc(map[string]simulation.ServiceFunc{
 				"bzz-sync": newSyncSimServiceFunc(nil),
-			})
+			}, false)
```

Comment:
why are these not true here,do we really want emerging and changing connections?